### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by gabzn**

### CODE REVIEW

File: client/src/components/Dashboard.js [hunk 1]
```diff
@@ -1,10 +1,68 @@
...
+// import { useNavigate, Navigate } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import jwt_decode from 'jwt-decode';
+import axios from 'axios';
 
 function Dashboard() {
+    const navigate = useNavigate();
+    const [bills, setBills] = useState();
+
+    useEffect(() => {
```
Comment (gabzn) [inline client/src/components/Dashboard.js:11]: Added display bills when the user is in the dashboard page.

File: client/src/components/Dashboard.js [hunk 2]
```diff
@@ -1,10 +1,68 @@
...
+                Click to go back to the log in page.
+            </button> */}
+
+            {/* Navigate can be used to check if the user is authed. If not, use it to redirect user back. */}
+            {/* <Navigate to='/login' /> */}
+
+            Welcome back!
+
+            Here is all your bills.
+            <div>{bills && bills.map(bill => bill.storeName)}</div>
```
Comment (gabzn) [inline client/src/components/Dashboard.js:65]: For now I only displayed the store names.

File: client/src/components/Login.js [hunk 3]
```diff
@@ -28,63 +24,45 @@ function Login() {
...
+    if(data.errors) {
+      setInvalidMessage(data.errors[0].msg);
+      setIsInvalid(true);
+    }
+
+    if(data.error) {
+      setInvalidMessage(data.error);
+      setIsInvalid(true);
+    }
+
```
Comment (gabzn) [inline client/src/components/Login.js:36]: Fixed login.

File: server/routes/billRoute.js [hunk 4]
```diff
@@ -102,4 +102,30 @@ router.post(
...
+	const userid = req.query.userid;
+	
+	try {
+		const bills = await Bill.find({hostID: userid});
+		res.json({bills});	
+	} catch (error) {
+		res.json({error});
+	}
+})
+
```
Comment (gabzn) [inline server/routes/billRoute.js:130]: Added route to get all the bills of an user.

Comment (WenxuC) [reply server/routes/billRoute.js:130]: why did you comment out my code?

Comment (gabzn) [reply server/routes/billRoute.js:130]: That's not your code. I wrote both routes but the one on the top didn't work and I forgot to take it out.

**Review by WenxuC — APPROVED** [review]

**Pull request merged**